### PR TITLE
Issue #26 Documentation NFS Volume

### PR DESCRIPTION
**Pull request by janleemans**
Detailing the documentation to set up a Single Instance DB using an NFS Volume
Signed-off-by: Jan Leemans <jan.leemans@oracle.com>

### CODE REVIEW

File: docs/sidb/README.md [hunk 1]
```diff
@@ -323,10 +323,17 @@ spec:
   persistentVolumeReclaimPolicy: Retain
   csi:
     driver: fss.csi.oraclecloud.com
-    volumeHandle: "<OCID of the file system>:<Mount Target>/<Export Path>"
+    volumeHandle: "<OCID of the file system>:<Mount Target IP Address>:/<Export Path>"
 ```
 
-**Note:** Whenever a mount target is provisioned in OCI, its `Reported Size (GiB)` values are very large. This is visible on the mount target page when logged in to the OCI console. Some applications will fail to install if the results of a space requirements check show too much available disk space. So specify, in gibibytes (GiB), the maximum capacity reported by file systems exported through this mount target. This setting does not limit the actual amount of data you can store.
+**Note:** 
+- Example volumeHandler in the above config file :
```
Comment (abhisbyk) [inline docs/sidb/README.md:330]: Shouldn't it be Example `volumeHandle`, right ?

Comment (janleemans) [reply docs/sidb/README.md]: Indeed, I'll correct

Comment (janleemans) [reply docs/sidb/README.md]: Corrected.

**Review by abhisbyk — APPROVED** [review]

**Comment by abhisbyk** [comment]
The doc changes look fine to me.

**Comment by yunus-qureshi** [comment]
Thanks @janleemans

**Pull request merged**